onnx_tf/handlers/frontend/size.py

```diff
@@ -27,10 +27,13 @@ def version_1(cls, node, **kwargs):
     if not need_cast:
       return [size_node]
 
+    node.attr['DstT'] = node.attr['out_type']
```

Comment:
I'm just curious, do we need any attributes other than 'DstT" here?

Reply:
If not, maybe we can just create a new attr object instead of re-using the node.attr, but add a new attr to it?

Reply:
We only need 'DstT'. So I created a new attrs now.